common/common.cpp

```diff
@@ -1702,27 +1705,27 @@ void llama_batch_add(
 
 #ifdef LLAMA_USE_CURL
 
-struct llama_model * llama_load_model_from_url(
-        const char * model_url,
-        const char * path_model,
-        const struct llama_model_params & params) {
-    // Basic validation of the model_url
-    if (!model_url || strlen(model_url) == 0) {
-        fprintf(stderr, "%s: invalid model_url\n", __func__);
-        return NULL;
+static std::string llama_download_hide_password_in_url(const std::string & url) {
```

Comment:
Since this function is used internally by `llama_download_file` for logging, maybe we can convert it to lambda function can place it inside `llama_download_file`?